src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+app.env.abacUrl=${ABAC_PDP_ENDPOINT_URL}
```

Comment:
Mangler properties her for context path, prometheus osv. Se eksempel i f.eks veilarboppfolging